### PR TITLE
Extension: Add MN dust sensor

### PR DESCRIPTION
**Pull request by microbit-mark**
Add dust sensor kit 

cc @eng-minnie

### CODE REVIEW

File: targetconfig.json [hunk 1]
```diff
@@ -171,7 +171,8 @@
             "CytronTechnologies/pxt-edubit",
             "MakeAndLearn/pxt-microshield",
             "4tronix/Orbit",
-            "elecfreaks/pxt-TPBot"
+            "elecfreaks/pxt-TPBot",
+            "https://github.com/eng-minnie/mn_dust"
```
Comment (abchatra) [inline targetconfig.json:175]: This is incorrect. You don't need github.com here.

**Comment by abchatra** [comment]
@eng-minnie Please reduce the png size to < 100kb. It is too big and will impact network.
https://github.com/eng-minnie/mn_dust/blob/master/icon.png

File: targetconfig.json (hunk 1 above)
Comment (microbit-mark) [reply targetconfig.json]: Ack. Fixed

**Comment by abchatra** [comment]
@eng-minnie your tags also look incorrect. Can you tag again with the latest build and update the pxt.json with that tag number? 

I can't test this package as I am unable to load it in makecode.

**Comment by microbit-mark** [comment]
I think this is now fixed @abchatra

**Comment by abchatra** [comment]
@microbit-mark I can't load this extension in makecode. Can you?

**Comment by eng-minnie** [comment]
I fixed it.
Can you check it?

**Comment by eng-minnie** [comment]
@abchatra @microbit-mark 
I fixed it.
Can you check it?

**Comment by microbit-mark** [comment]
I can't load in makecode @eng-minnie. Not sure why as the code from https://github.com/eng-minnie/mn_dust/blob/master/dustkit.ts works in the editor.

Console output:
Screenshot 2020-06-04 at 13 04 12

**Comment by abchatra** [comment]
There are no releases in this extension. There are no new blocks or API in this extension either. I am closing this.

**Pull request closed without merging**